package.json

```diff
@@ -59,7 +59,7 @@
     "@angular/core": "13.0.0-next.2",
     "@angular/forms": "13.0.0-next.2",
     "@angular/platform-browser": "13.0.0-next.2",
-    "@types/googlemaps": "^3.43.1",
+    "@types/google.maps": "^3.45.6",
```

Comment:
Is there more information we can include in the commit message about these updates types?

Reply:
I haven't looked much into it, but the old package is marked as deprecated on npm and doesn't even show up in searches anymore.